sixtyfps_compiler/widgets/fluent/sixtyfps_widgets_impl.60

```diff
@@ -120,33 +120,35 @@ ScrollBar := Rectangle {
    // border-color: Palette.button-background;
     border-width: 1px;
     property <bool> horizontal;
-    property<length> max;
+    property<length> maximum;
```

Comment:
Makes sense for consistency with the other widgets (and ScrollBar seems internal, so no problem), but we have an inconsistency here in the sense that the widgets use minimum/maximum and the layout uses min/max.

Reply:
This is an internal api only.

The problem is that i can't use a property called `max` or i can't call the function `max(..., ...)`
See also https://github.com/sixtyfpsui/sixtyfps/issues/273 currently, functions resolves last.